recipes/DAVE/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "useDAVE" %}
+{% set version = "0.1.0" %}                         
+package:
+  name: dave
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e3ca64989027fa7ebcb7a346864f2e84a5fcf90106aec897b774be1111d6b32a
+ 
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - setuptools
+    - pyscaffold
+  run:
+    - python >=3.5
+    - setuptools
+    - pyscaffold
+    - vtk >=8.2.0
+    - ipython
+    - pyside2
+    - numpy
+    - scipy
+    - matplotlib
```

Comment:
Can this be `matplotlib-base?`

Reply:
Checked and matplotlib-base works as well. Did not know it existed.... Thanks!